src/components/input/index.jsx

```diff
@@ -84,11 +85,17 @@ class Input extends React.Component {
         this.setValue(e.currentTarget.value)
     }
 
+    click(e) {
```

Comment:
i need the click because it is awkward with just focus, if the input is already focused and you try clicking on it, nothing happens